src/components/component-collection/ComponentCollection.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import styled from "styled-components";
+import { ThemeProvider } from "styled-components";
+
+const Container = styled.div`
+  width: 100%;
+  height: 200px;
+  padding: 20px 20px 20px 40px;
+  background-color: ${props => props.theme.colors.primaryPink};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    height: 280px;
+    padding: 30px 10px 10px 10px;
+  }
+`;
+
+const GownImageMobile = styled.img`
+  width: 160px;
+  height: 150px;
```

Comment:
Try taking the height and width directly from the figma for this one and the desktop version